client/core/simnet_trade.go

```diff
@@ -1998,6 +2054,16 @@ func (s *simulationTest) assertBalanceChanges(client *simulationClient) error {
 		}
 	}
 
+	// Account assets require a refund fee in addition to the swap amount.
+	if isRefund {
+		if ord.Sell && accountBIPs[s.base.id] {
+			baseFees = int64(dexeth.RefundGas(0 /*version*/)) * 2
```

Comment:
So here we have the fee rate as 2 gwei/gas.  Is that generally expected or dictated for eth simnet? Can we doc?

Also, I think we do want to account for refund fees for UTXO assets too.  Perhaps we can just approximate the refund fee as the same as the swap fee for UTXO assets, given the rather large [0.95, 1.05] buffer we add below.

Or do we need to this info to the `FeeBreakdown`?

Reply:
> So here we have the fee rate as 2 gwei/gas. Is that generally expected or dictated for eth simnet?

It's just expected from simnet. It is the tip cap. The, whats it called, consensus enforced fee is much smaller. I think I've seen that fee rise above a few gwei after running loadbot for a while though.

> Or do we need to add this info to the FeeBreakdown?

Maybe.

Reply:
Added utxo refunds, based off of the btc price of a refund.